src/main/webapp/app/exam/manage/exams/exam-update.component.html

```diff
@@ -60,67 +73,43 @@ <h4>{{ 'artemisApp.examManagement.editExam' | artemisTranslate }}</h4>
                         [(ngModel)]="exam.endDate"
                         [error]="!isValidEndDate"
                         name="endDate"
                         id="endDate"
                     ></jhi-date-time-picker>
                 </div>
             </div>
-
-            <div class="row mb-3">
-                <div class="col-sm-4">
-                    <jhi-date-time-picker
-                        labelName="{{ 'artemisApp.examManagement.publishResultsDate' | artemisTranslate }}"
-                        [(ngModel)]="exam.publishResultsDate"
-                        [error]="!isValidPublishResultsDate"
-                        name="publishResultsDate"
-                        id="publishResultsDate"
-                    ></jhi-date-time-picker>
-                </div>
-                <div class="col-sm-4">
-                    <jhi-date-time-picker
-                        labelName="{{ 'artemisApp.examManagement.examStudentReviewStart' | artemisTranslate }}"
-                        [(ngModel)]="exam.examStudentReviewStart"
-                        [error]="!isValidExamStudentReviewStart"
-                        name="examStudentReviewStart"
-                        id="examStudentReviewStart"
-                    ></jhi-date-time-picker>
-                </div>
-                <div class="col-sm-4">
-                    <jhi-date-time-picker
-                        labelName="{{ 'artemisApp.examManagement.examStudentReviewEnd' | artemisTranslate }}"
-                        [(ngModel)]="exam.examStudentReviewEnd"
-                        [error]="!isValidExamStudentReviewEnd"
-                        name="examStudentReviewEnd"
-                        id="examStudentReviewEnd"
-                    ></jhi-date-time-picker>
-                    <span *ngIf="!isValidExamStudentReviewEnd" class="invalid-feedback"> {{ 'artemisApp.examManagement.reviewDatesInvalidWarning' | artemisTranslate }}</span>
-                </div>
-            </div>
-
             <div class="row mb-3">
                 <div class="col-sm-6">
-                    <label class="form-check-label" for="gracePeriod" jhiTranslate="artemisApp.examManagement.gracePeriod">Grace period (seconds)</label>
-                    <input id="gracePeriod" name="gracePeriod" class="form-control" type="number" min="0" [(ngModel)]="exam.gracePeriod" />
-                </div>
-                <div class="col-sm-6">
-                    <label class="form-check-label" for="numberOfCorrectionRoundsInExam" jhiTranslate="artemisApp.examManagement.numberOfCorrectionRoundsInExam">
-                        Number of correction rounds in Exam
-                    </label>
+                    <label class="form-check-label" for="workingTimeInMinutes" jhiTranslate="artemisApp.examManagement.workingTime">Working Time</label>
+                    <jhi-help-icon-without-translation
```

Comment:
I might miss something important here, but why does not `<jhi-help-icon>`  work here?

Reply:
The tooltip is to be changed based on the selected mode. 
For RealExams the working time is callculated based on the start- and end-Date and therefore does not need to be inputted by the user. For TestExams, the working time has to be inputted by the user. That's why the `without-translation` was used here to switch between the Strings.

Reply:
I also don't really see, why you can't just use `jhi-help-icon` here.
You could simply do something like this:
```html
<jhi-help-icon
[text]="{{ 'artemisApp.examManagement' + (exam.testExam ? '.testExam' : '') + '.workingTimeTooltip' }}"
></jhi-help-icon>
```
as `jhi-help-icon-without-translation` should only be used if you actually do not want to translate the text, which you do in this case.